package.json

```diff
@@ -27,8 +28,10 @@
     "json-bigint": "https://github.com/Caltech-IPAC/json-bigint",
     "local-storage": "~2.0",
     "lodash": "~4.17",
+    "mathjax-full": "^3.2.2",
```

Comment:
Installing `better-react-mathjax` will also pull in `mathjax-full` as dependency.  Do we need to explicitly declare here?

Reply:
Yes because of `import mathjaxSrc from 'mathjax-full/es5/tex-svg.js?url';` in `LatexUtil.jsx` otherwise linter complains that this node module is not found.